### ISSUE TITLE
Datadog Real User Monitoring breaks node express client server on IE11

### ISSUE THREAD

**Issue by lauraeci** [DataDog/browser-sdk#611]
Using the @datadog/browser-rum package with Node apollo web client fails on IE11.

Seeing multiple requests to https://run-http-intake.logs.datadoghq.com on IE11 that are different than Chrome. Unable to login to the application on IE11. Working theory is that datadogRUM is blocking other application requests on IE11. When datadogRUM is removed, the application is working correctly.

Chrome and IE11 requests:

- Request Method: POST RequestURL: https://rum-http-intake.logs.datadoghq.com/v1/input/pub{id}?_dd.application_id={id}&ddsource=browser&&ddtags=sdk_version:1.25.2.env:local&batch_time={timestamp}

Only IE11 request:

- 
Request Method: CONNECT RequestURL: https://rum-http-intake.logs.datadoghq.com Proxy-Connection: Keep-Alive

Using the init method from the documentation like: 
```
import { datadogRum } from '@datadog/browser-rum' 

datadogRum.init({
          applicationId: '<DATADOG_APPLICATION_ID>',
          clientToken: '<DATADOG_CLIENT_TOKEN>',
           site: '<DATADOG_SITE>',
           env: process.env.ENVIRONMENT,
            silentMultipleInit: true,
            sampleRate: 100,
            trackInteractions: false,
            useSecureSessionCookie: true,
})
```
Please let me know what additional information I should find to help with this issue. IE11 issues are no fun. Thank you!

**Comment by bcaudan** [comment]
Hi @lauraeci,

Thank you for raising that.

- Are you application requests triggered correctly?
- Do you see any network or console error associated to this issue? 
- Can I have a look at your setup somewhere?

**Comment by lauraeci** [comment]
Hi @bcaudan,

Thank you for your help. 

- We are getting logs uploaded to RUM Explorer from IE11. No network or console errors, so I'm guessing an application request is not happening for login. 
- I can teach out to our live support to provide more detailed information about our setup. Please let me know what information I can provide to them to facilitate the request. 
- Reaching out to our team to provide more details on if our application requests are triggered correctly.

**Comment by bcaudan** [comment]
Is there a public page where I can have a look at your application?

**Comment by carlpeaslee** [comment]
Hi @bcaudan , thanks for looking into this. 

> Are you application requests triggered correctly?
We are using apollo client for all our outgoing requests. We aren't having any problems on IE elsewhere. Our configuration isn't especially unusual though we are polyfilling fetch with `whatwg-fetch`
> Do you see any network or console error associated to this issue?
We don't. It doesn't seem as though the network requests are even sent. 
> Can I have a look at your setup somewhere?
Which aspect of our setup would you like to look at? This is how we're initializing datadogRUM 

```typescript
    datadogRum.init({
        applicationId: /*REDACTED*/,
        // this token is passed to the client, so not secret.
        // https://docs.datadoghq.com/account_management/api-app-keys/#client-tokens
        clientToken: /*REDACTED*/,
        site: 'datadoghq.com',
        env: process.env.ENVIRONMENT, //we are using webpack to inject process.env vars into our client app
        sampleRate: 100,
        trackInteractions: true,
    })
```

**Comment by bcaudan** [comment]
Hi @carlpeaslee,

Thanks for all these details!
I have been able to reproduce the issue on IE 11 on your demo application.

The issue is due to a mismatch between browser-rum and browser-logs version.
I would advise you to either:

- install browser-rum@1.12.9 to match browser-logs version in your setup
- upgrade your browser-logs version to match your browser-rum version

I have tested with browser-rum@1.12.9 and the issue does not happen anymore.
